README.md

```diff
@@ -12,8 +12,7 @@ reading out data from Modbus devices. The design includes:
   them
 * output array. The responses from Modbus devices are stored in it, as
   commanded (readout command)
-* output to telemetry FIFO, with commands to update telemetry memory with
-  values readout from device
+* output to telemetry FIFO register values, etc

```

Comment:
etc --> etc.

Reply:
Thanks. Changed this and all following problems.